src/common/exif.cc

```diff
@@ -154,11 +154,14 @@ static void _get_xmp_tags(const char *prefix, GList **taglist)
   }
 }
 
-GList *dt_exif_get_exiv2_taglist()
+static GList *taglist;
```

Comment:
You're missing a = NULL there otherwise we will be getting random memory value and possible crash.

Reply:
I also think that such static persistent storage variable should be placed at the start of the file with comments about the persistency and the fact that it is initialize early in dt_init()

Reply:
I had read that static and global variable were always initialized to 0 or NULL...
But you are right, it's more secure with an explicit value.
I'll do